fluentd/configs.d/openshift/es-copy-config.conf

```diff
@@ -17,9 +17,9 @@
       # recreate/reload connections
       reload_connections false
       reload_on_failure false
-      flush_interval 5s
-      max_retry_wait 300
-      disable_retry_limit true
+      flush_interval "#{ENV['ES_FLUSH_INTERVAL'] || '5s'}"
+      max_retry_wait "#{ENV['ES_RETRY_WAIT'] || '300'}"
+      disable_retry_limit "#{ENV['ES_DISABLE_RETRY_LIMIT'] || 'true'}"
```

Comment:
@portante I assume you're talking about disable_retry_limit? I'll leave it hard coded as `true`.